intro-to-R-tidyverse/00a-rstudio_guide.Rmd

```diff
@@ -9,11 +9,12 @@ output:
 ---
 
 Our course material uses [R Markdown Notebooks](https://bookdown.org/yihui/rmarkdown/notebook.html).
-Opening and running R Markdown (`.Rmd`) files requires [RStudio](https://www.rstudio.com/products/RStudio/).
+Opening and running R Markdown (`.Rmd`) files requires [RStudio](https://posit.co/products/open-source/rstudio/).
 RStudio is an [integrated development environment](https://en.wikipedia.org/wiki/Integrated_development_environment) or IDE that makes it easier to program in R.
 **Here, our objective is to provide a brief guide to getting started with RStudio.**
 
-Note there is an excellent _Take a tour of RStudio’s IDE_ video from RStudio [here](https://www.rstudio.com/products/rstudio/).
+You may find this handy [RStudio IDE Cheatsheet](https://rstudio.github.io/cheatsheets/html/rstudio-ide.html) useful too!
```

Comment:
Maybe linking to the docs (and their getting started page) would be better? 
https://docs.posit.co/ide/user/ide/get-started/

Seems like a more gentle/less busy introduction

Reply:
good find!!